import_export/formats/base_formats.py

```diff
@@ -173,7 +173,7 @@ def create_dataset(self, in_stream):
         """
         from io import BytesIO
         import openpyxl
-        xlsx_book = openpyxl.load_workbook(BytesIO(in_stream), read_only=True)
+        xlsx_book = openpyxl.load_workbook(BytesIO(in_stream), read_only=True, data_only=True)
```

Comment:
Could yo explain the ramifications of this change? Would it have any effect on the users experience?

Reply:
Instead of getting a formula on import now you get the value of that formula.